src/main/resources/db/changelog/changelogs/init.sql

```diff
@@ -0,0 +1,6 @@
+---TODO create parameter for service name
+create table if not exists template (
```

Comment:
Minor - does it make sense to use `if not exists` if we use liquibase?

Reply:
fixed